src/main/java/dao/UserDao.java

```diff
@@ -1,4 +1,74 @@
 package dao;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import model.User;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
 public class UserDao {
+    public static String usersJsonFile = "users.json";
+
+    public static List<User> users = new ArrayList<User>() {{
```

Comment:
use factory method from Java 9 List.of(obj,obj,obj);

Reply:
fixed. use List.of instead of new ArrayList<User>()